core/src/main/java/de/danoeh/antennapod/core/util/playback/PlaybackController.java

```diff
@@ -70,6 +72,14 @@ public synchronized void init() {
         }
     }
 
+    public boolean isRefresed() {
+        return refresed;
```

Comment:
It looks like this is a leftover from an earlier experiment. The method is never used.